.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
This works fine. But, my suggestion would be to use `1.` in all the list elements, and Markdown will actually use the correct numbers. This way, if you want to add another bullet in between you don't have to worry about the enumeration.